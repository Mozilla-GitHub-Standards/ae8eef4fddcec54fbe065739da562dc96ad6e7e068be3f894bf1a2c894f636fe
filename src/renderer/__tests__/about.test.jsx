/** @format */

import React from 'react'
import Renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router-dom'

import About from '../containers/AboutPage'

test('Page: About', () => {
  const component = Renderer.create(
    <MemoryRouter>
      <About />
    </MemoryRouter>
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
