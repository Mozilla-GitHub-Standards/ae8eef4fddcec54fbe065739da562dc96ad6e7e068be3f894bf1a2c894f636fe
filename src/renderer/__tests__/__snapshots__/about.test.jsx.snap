// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page: About 1`] = `
<div>
  <p>
    Sample project for an Electron based application.
  </p>
  <div
    className="MuiBottomNavigation-root-2 AboutPage-root-1"
  >
    <a
      className="MuiButtonBase-root-8 MuiBottomNavigationAction-root-3 MuiBottomNavigationAction-selected-4"
      href="/"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role="button"
      tabIndex="0"
    >
      <span
        className="MuiBottomNavigationAction-wrapper-6"
      >
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root-11"
          focusable="false"
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M10 20v-6h4v6h5v-8h3L12 3 2 12h3v8z"
          />
          <path
            d="M0 0h24v24H0z"
            fill="none"
          />
        </svg>
        <span
          className="MuiBottomNavigationAction-label-7 MuiBottomNavigationAction-selected-4"
        >
          Home
        </span>
      </span>
      <span
        className="MuiTouchRipple-root-20"
      />
    </a>
    <a
      className="MuiButtonBase-root-8 MuiBottomNavigationAction-root-3"
      href="/preferences"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role="button"
      tabIndex="0"
    >
      <span
        className="MuiBottomNavigationAction-wrapper-6"
      >
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root-11"
          focusable="false"
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M0 0h20v20H0V0z"
            fill="none"
            transform="scale(1.2, 1.2)"
          />
          <path
            d="M15.95 10.78c.03-.25.05-.51.05-.78s-.02-.53-.06-.78l1.69-1.32c.15-.12.19-.34.1-.51l-1.6-2.77c-.1-.18-.31-.24-.49-.18l-1.99.8c-.42-.32-.86-.58-1.35-.78L12 2.34c-.03-.2-.2-.34-.4-.34H8.4c-.2 0-.36.14-.39.34l-.3 2.12c-.49.2-.94.47-1.35.78l-1.99-.8c-.18-.07-.39 0-.49.18l-1.6 2.77c-.1.18-.06.39.1.51l1.69 1.32c-.04.25-.07.52-.07.78s.02.53.06.78L2.37 12.1c-.15.12-.19.34-.1.51l1.6 2.77c.1.18.31.24.49.18l1.99-.8c.42.32.86.58 1.35.78l.3 2.12c.04.2.2.34.4.34h3.2c.2 0 .37-.14.39-.34l.3-2.12c.49-.2.94-.47 1.35-.78l1.99.8c.18.07.39 0 .49-.18l1.6-2.77c.1-.18.06-.39-.1-.51l-1.67-1.32zM10 13c-1.65 0-3-1.35-3-3s1.35-3 3-3 3 1.35 3 3-1.35 3-3 3z"
            transform="scale(1.2, 1.2)"
          />
        </svg>
        <span
          className="MuiBottomNavigationAction-label-7"
        >
          Settings
        </span>
      </span>
      <span
        className="MuiTouchRipple-root-20"
      />
    </a>
  </div>
</div>
`;
